### PR TITLE
Update handlers to support new FileFormat property for Faq.

### PR DESCRIPTION
**Pull request by kevinawskendra**
**Notes**
aws-kendra-faq
- Updated Kendra SDK version to `2.14.21`.
- Added FileFormat property to Faq resource and updated handlers.

**Testing**
- Unit tests

### CODE REVIEW

File: aws-kendra-faq/aws-kendra-faq.json [hunk 1]
```diff
@@ -46,6 +46,15 @@
       "minLength": 1,
       "maxLength": 1000
     },
+    "FileFormat": {
```
Comment (angranamzn) [inline aws-kendra-faq/aws-kendra-faq.json:49]: np; I'd probably name this `FaqFileFormat` so it's clear this is only for FAQ as I think the generated enum in the SDK is called.

**Review by angranamzn — DISMISSED** [review]
Looks good.

**Review by waquarahmad515 — APPROVED** [review]

File: aws-kendra-faq/aws-kendra-faq.json (hunk 1 above)
Comment (kevinawskendra) [reply aws-kendra-faq/aws-kendra-faq.json:49]: Good point about the enum name. This is in the FAQ provider so it should be clear it is for FAQ. Also the field in CreateFaqRequest is named FileFormat.

File: aws-kendra-faq/src/test/java/software/amazon/kendra/faq/CreateHandlerTest.java [hunk 2]
```diff
@@ -74,6 +74,7 @@ public void handleRequest_SimpleSuccess() {
         String name = "name";
         String s3Key = "s3Key";
         String s3Bucket = "s3Bucket";
+        String fileFormat = "CSV";
```
Comment (EricCeres) [inline aws-kendra-faq/src/test/java/software/amazon/kendra/faq/CreateHandlerTest.java:77]: As fileFormat is optional do we have a test for that case?

File: aws-kendra-faq/src/test/java/software/amazon/kendra/faq/UpdateHandlerTest.java [hunk 3]
```diff
@@ -73,6 +73,7 @@ public void handleRequest_SimpleSuccess() {
         String name = "name";
         String s3Key = "s3Key";
         String s3Bucket = "s3Bucket";
+        String fileFormat = "CSV";
```
Comment (EricCeres) [inline aws-kendra-faq/src/test/java/software/amazon/kendra/faq/UpdateHandlerTest.java:76]: Same as above do we have case where this is not provided?

File: aws-kendra-faq/src/test/java/software/amazon/kendra/faq/CreateHandlerTest.java (hunk 2 above)
Comment (kevinawskendra) [reply aws-kendra-faq/src/test/java/software/amazon/kendra/faq/CreateHandlerTest.java:77]: Added new test.

File: aws-kendra-faq/src/test/java/software/amazon/kendra/faq/UpdateHandlerTest.java (hunk 3 above)
Comment (kevinawskendra) [reply aws-kendra-faq/src/test/java/software/amazon/kendra/faq/UpdateHandlerTest.java:76]: Added new test.

**Review by EricCeres — APPROVED** [review]

**Review by angranamzn — APPROVED** [review]

**Review by swighe — APPROVED** [review]

**Pull request merged**